vendor/bat-native-confirmations/src/bat/confirmations/internal/confirmations_create_confirmation_request_unittest.cc

```diff
@@ -12,6 +12,8 @@
 #include "bat/confirmations/internal/confirmations_impl.h"
 #include "bat/confirmations/internal/create_confirmation_request.h"
 #include "bat/confirmations/internal/security_helper.h"
+#include "bat/confirmations/internal/platform_helper.h"
```

Comment:
sorting

Reply:
Raised https://github.com/brave/brave-browser/issues/9166 to resolve across Brave Ads